flowdb/sql/04_schema_other.sql

```diff
@@ -68,3 +68,15 @@ Schema used for temp storage during etl.
 */
 
 CREATE SCHEMA IF NOT EXISTS etl;
+
+CREATE TYPE cdrtype AS ENUM ('voice', 'sms', 'mds');
+CREATE TYPE status AS ENUM ('in_process','done', 'quarantine');
+CREATE TABLE etl.etl (
+        id SERIAL NOT NULL,
+        file_name VARCHAR,
+        cdr_type cdrtype,
+	cdr_date DATE,
```

Comment:
Nitpick (indentation / tab?):
```suggestion
      cdr_date DATE,
```

Reply:
I think this is a github bug...As you can see your suggestion is no different...

Reply:
It is... I replaced a tab with a bunch of spaces ;)

Reply:
Yeah but look at https://github.com/Flowminder/FlowKit/blob/cb126150144d29bdc6181af8ec3e6771e1fb9bae/flowdb/sql/04_schema_other.sql no weird indent?